dcat/index.html

```diff
@@ -117,8 +117,8 @@ <h2>Introduction</h2>
     </p>
 
     <p>
-        Data is often provided through a service accessed through a form or API which supports selection of an extract, sub-set, or combination of data.
-        DCAT allows the description of a data access service to be included in a catalog.
+        Data is often provided through a service accessed through an API, which supports selection of extracts, sub-sets, or combinations of data.
```

Comment:
Whoops. I thought 'a form' had been removed. Must have got dropped in a merge conflict somewhere.